### PR TITLE
Add automation script for making supported devices table

### PR DESCRIPTION
**Pull request by ethanaobrien**
Written in javascript, since its what I'm familiar with

This pr also
- Fixes a typo on the debugging page (`imtel` -> `intel`)
- Adds a note that notifies users that unofficial versions of windows are not supported

### CODE REVIEW

**Review by ninelore — CHANGES_REQUESTED** [review]
I'd go for a json file with the devices instead of inline. Makes it les cluttered and easier to edit

**Comment by WeirdTreeThing** [comment]
Maybe we could scrape this data from somewhere?

**Comment by ethanaobrien** [comment]
>Maybe we could scrape this data from somewhere?

I thought about it, but where would we scrap it from? We'd need to scrap windows/linux/and macos support, as well as specific device notes.

**Comment by ethanaobrien** [comment]
>I'd go for a json file with the devices instead of inline. Makes it less cluttered and easier to edit

This change has been implemented.

I also wrote a how to edit example at the top of the index.js file (see https://github.com/ethanaobrien/docz/blob/275ee6b5ac61a5efe5584966029ad34d68005f68/supported-devices/index.js#L6)

To reduce clutter, I have also added "default" options for each generation (which also makes editing much easier)

**Review by mine-man3000 — COMMENTED** [review]
looks fine to me (although I did skim through the JSON)

File: docs/installing-windows.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ nav_order: 11
 {: .warning }
 Do not use Windows 8 or older. Only recent Windows 10 and 11 builds are supported.
 
+{: .warning }
+Do not use unofficial versions of Windows, such as ghost spectre or tiny1X. They are not supported.
```
Comment (meghan06) [inline docs/installing-windows.md:18]: >such as ghost spectre or tiny1X. They are not supported.

Should be 

such as Ghost Specter or tiny10/11. They are not supported and may contain **undetectable** malware and/or security vulnerabilities.

**Review by meghan06 — CHANGES_REQUESTED** [review]

File: docs/installing-windows.md (hunk 1 above)
Comment (ethanaobrien) [reply docs/installing-windows.md]: Fixed~

**Review by meghan06 — APPROVED** [review]
Updated documentation looks good to me!

idk about the rest lol

**Review by ninelore — APPROVED** [review]

**Pull request merged**